### PR TITLE
Add in curated list of blockchain-related academic papers.

### PR DESCRIPTION
**Pull request by WizardOfAus**
**https://github.com/decrypto-org/blockchain-papers**

**A curated list of blockchain-related academic papers**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -484,6 +484,7 @@
 ## Decentralized Systems
 
 - [Bitcoin](https://github.com/igorbarinov/awesome-bitcoin) - Bitcoin services and tools for software developers.
+- [Blockchain Papers](https://github.com/WizardOfAus/blockchain-papers) - Curated list of blockchain-related academic papers.
```
Comment (legodude17) [inline readme.md]: Describe the topic, not the list.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/decrypto-org/blockchain-papers
✖ Linting

  README.md:8:1
  ✖    1:1    Missing or invalid Table of Contents                                                                                                                                                                                                                                                remark-lint:awesome/toc
  ✖    1:22   Invalid badge source                                                                                                                                                                                                                                                                remark-lint:awesome/badge
  ✖    7:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖    8:1    Marker should be 2, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖    8:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖    9:1    Marker should be 3, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖    9:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   10:1    Marker should be 4, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   10:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   11:1    Marker should be 5, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   11:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   12:1    Marker should be 6, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   12:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   13:1    Marker should be 7, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   13:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   14:1    Marker should be 8, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   14:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   15:1    Marker should be 9, was 1                                                                                                                                                                                                                                                           remark-lint:ordered-list-marker-value
  ✖   15:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   16:1    Marker should be 10, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   16:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   17:1    Marker should be 11, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   17:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   18:1    Marker should be 12, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   18:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   19:1    Marker should be 13, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   19:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   20:1    Marker should be 14, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   20:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   21:1    Marker should be 15, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   21:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   22:1    Marker should be 16, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   22:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   23:1    Marker should be 17, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   23:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   24:1    Marker should be 18, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   24:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   25:1    Marker should be 19, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   25:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   26:1    Marker should be 20, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   26:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   27:1    Marker should be 21, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   27:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   28:1    Marker should be 22, was 1                                                                                                                                                                                                                                                          remark-lint:ordered-list-marker-value
  ✖   28:4    Invalid list item link URL                                                                                                                                                                                                                                                          remark-lint:awesome/list-item
  ✖   34:3    Invalid list item link                                                                                                                                                                                                                                                              remark-lint:awesome/list-item
  ✖   36:3    Link to http://delivery.acm.org/10.1145/3140000/3132259/p36-narayanan.pdf?ip=88.197.32.177&id=3132259&acc=OPEN&key=5641A0C343C36AC1%2E4ABAF8470B13FB25%2E4D4702B0C3E38B35%2E6D218144511F3437&__acm__=1541082825_ead879dbf5b3644bfed2a09e11b47834 is dead                            remark-lint:no-dead-urls
  ✖   36:266  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   41:108  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   43:99   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   45:137  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   49:125  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   50:91   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   51:113  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   56:133  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖   57:15   Link to http://www.cs.huji.ac.il/~avivz/pubs/15/btc_ghost_full.pdf is dead                                                                                                                                                                                                          remark-lint:no-dead-urls
  ✖   73:3    Invalid list item link                                                                                                                                                                                                                                                              remark-lint:awesome/list-item
  ✖   76:106  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  111:146  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  112:185  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  113:102  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  114:176  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  124:96   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  125:149  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  126:140  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  142:114  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  147:3    Link to http://newspaper23.com/ripped/2014/11/http-_____-___-_www___-blockstream___-com__-_sidechains.pdf is dead                                                                                                                                                                   remark-lint:no-dead-urls
  ✖  147:158  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  155:82   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  160:155  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  164:3    Link to https://liquidity.network/NOCUST_Liquidity_Network_Paper.pdf is dead                                                                                                                                                                                                        remark-lint:no-dead-urls
  ✖  164:124  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  165:93   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  166:117  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  169:104  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  183:132  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  188:98   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  194:3    Link to http://delivery.acm.org/10.1145/2980000/2978326/p270-zhang.pdf?ip=46.176.188.9&id=2978326&acc=OA&key=4D4702B0C3E38B35%2E4D4702B0C3E38B35%2E4D4702B0C3E38B35%2E594C525CFFA2AFAF&CFID=923932938&CFTOKEN=56121949&__acm__=1492299159_38039f3afa858f241818fdcf190e0200 is dead  remark-lint:no-dead-urls
  ✖  194:323  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  205:96   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  212:96   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  215:163  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  216:109  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  217:130  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  218:59   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  219:106  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  231:75   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  232:3    Link to https://storj.io/storj.pdf is dead                                                                                                                                                                                                                                          remark-lint:no-dead-urls
  ✖  232:75   List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  233:138  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item
  ✖  234:112  List item link and description must be separated with a dash                                                                                                                                                                                                                        remark-lint:awesome/list-item

  91 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: @WizardOfAus :arrow_up:

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**